packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -175,7 +178,12 @@ const makeParamManagerBuilder = zoe => {
   };
 
   // Invitations are closely held, so we should only reveal the amount publicly.
+  // ??? how is this guaranteed: ?
```

Comment:
The approach here makes it possible for contracts to get the actual invitation privately, and legibly assure clients that it matches the publicly visible invitation amount. Contract reviewers still have to manually verify that the actual invitation is handled carefully.

Reply:
11b7d32ba